samples/js/fb/find-photos/src/main.js

```diff
@@ -22,7 +22,7 @@ import {
 const args = argv
   .usage(
     'Finds photos in slurped Facebook metadata\n\n' +
-    'Usage: fb/find-photos <in-object> <out-dataset>')
+    'Usage: noms-facebook-find-photos <in-dataset> <out-dataset>')
```

Comment:
Is this correct? We do not have a bin section in package.json so there is no binary installed.l To run this we need to do:

```sh
node . <in-dataset> <out-dataset>
```